### ISSUE TITLE
HTML Indenting

### ISSUE THREAD

**Issue by cyouh95** [justamanda/prj-rev-bwfs-dasmoto#1]
Try to have consistent indenting in your document and use indents only for nested elements. You could instead add some linebreaks between the blocks of code if you'd like to improve readability!

https://github.com/justamanda/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L9-L21

**Comment by justamanda** [comment]
Thank you, I was actually a bit confused on that. So all of the headers just live under the body or div tag? I'm confused about which things count as children and which as siblings, but I suppose all of the numbered headers should be siblings, huh?

**Comment by cyouh95** [comment]
Yup! so anything *between the open and close tags* of an element are the *children* of that element, and anything kind of just above or below it are the siblings (ie. `head` & `body` are siblings, as are `h2` & `p`. Then, `h2` & `p` are also the children of `body`):

```html
<head>
    ...
</head>
<body>
    <h2> ... </h2>
    <p> ... </p>
</body>
```
